caluma/workflow/jexl.py

```diff
@@ -6,6 +6,12 @@
 from ..core.jexl import ExtractTransformSubjectAnalyzer
 
 
+class GroupJexl(JEXL):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.add_transform("groups", lambda spec: spec)
```

Comment:
This transform doesn't really do anything.. is this correct? Or shouldn't it actually *do* something? (I see that some other transforms do the same, but still..

Reply:
On another note regarding "task" vs "tasks" (plural) transforms below: I'd suggest validating the input to be an actual list in the plural transform variant(s). Otherwise you (or the user) will get a nasty surprise if the plural-form is used on a singular input (or vice versa)

Reply:
Those transforms are used as markers. This way a jexl expression can be analyzed and groups resp. tasks extracted without knowing the input data.

The input type of a transform should be validated I agree. I will add this to the respective validator.

Reply:
Added additional validation for transform subject types.